StripeConnect/StripeConnect/Source/Internal/Webview/ConnectWebView.swift

```diff
@@ -14,11 +15,14 @@ import WebKit
  - Camera access
  - Popup windows
  - Opening email links
- - Downloads  TODO MXMOBILE-2485
+ - Downloads 
  */
 @available(iOS 15, *)
 class ConnectWebView: WKWebView {
 
+    /// File URL for a downloaded file
+    var downloadedFile: URL?
+
```

Comment:
@mxl-stripe do we need to be robust to potentially handle multiple files that simultaneous download?

The current design only stores one file at a time. If there's multiple downloads that occur simultaneously, we could get some unexpected behavior.

If we need to, we could potentially handle multiple files, but it would take some effort. We'd have to effectively make a queue of downloaded files and wait for them to all finish before showing the preview.

Reply:
I don't think we should ever be triggering multiple downloads at the same time. Is it easy to detect if that is happening? I wonder if we could add a metric for that situation and monitor that it's zero.

Reply:
Yes, we should be able to add a metric for this – I added a comment to [MXMOBILE-2491](https://jira.corp.stripe.com/browse/MXMOBILE-2491) so I remember to do that when adding analytics :-)